scripts/trigger-mobile-metrics.py

```diff
@@ -45,43 +48,42 @@ def TriggerJob(token, commit, job):
 
     data = {
         "build_parameters": {
-          "CIRCLE_JOB": job,
-          "BENCHMARK_COMMIT": commit
+            "CIRCLE_JOB": job,
+            "BENCHMARK_COMMIT": commit
         }
     }
 
     response = requests.post(url, auth=(token, ""), headers=headers, json=data)
 
     if response.status_code != 201 and response.status_code != 200:
-      print("Error triggering the CircleCI: %s." % response.json()["message"])
-      sys.exit(1)
+        print("Error triggering the CircleCI: %s." % response.json()["message"])
+        sys.exit(1)
     else:
-      response_dict = json.loads(response.text)
-      build_url = response_dict['build_url']
-      print("Started %s: %s" % (job, build_url))
-
-def Main():
-  token = os.getenv("MOBILE_METRICS_TOKEN")
-  commit = os.getenv("CIRCLE_SHA1")
-  
-  if token is None:
-    print("Error triggering because MOBILE_METRICS_TOKEN is not set")
-    sys.exit(1)
-
-  # Publish results that have been committed to the main branch.
-  # Development runs can be found in CircleCI after manually triggered.
-  publishResults = os.getenv("CIRCLE_BRANCH") == "main"
-  TriggerWorkflow(token, commit, publishResults)
-
-  if publishResults:
-    TriggerJob(token, commit, "android-navigation-benchmark")
-    TriggerJob(token, commit, "android-navigation-code-coverage")
-    TriggerJob(token, commit, "android-navigation-binary-size")
-  else:
-    TriggerJob(token, commit, "android-navigation-code-coverage-ci")
-    TriggerJob(token, commit, "android-navigation-binary-size-ci")
-
-  return 0
+        response_dict = json.loads(response.text)
+        build_url = response_dict['build_url']
+        print("Started %s: %s" % (job, build_url))
+
+
+def main():
+    token = os.getenv("MOBILE_METRICS_TOKEN")
+    commit = os.getenv("CIRCLE_SHA1")
+
+    if token is None:
+        print("Error triggering because MOBILE_METRICS_TOKEN is not set")
+        sys.exit(1)
+
+    # Publish results that have been committed to the main branch.
+    # Development runs can be found in CircleCI after manually triggered.
+    publish_results = os.getenv("CIRCLE_BRANCH") == "main"
+
+    trigger_workflow(token, commit, publish_results)
+
+    if not publish_results:
+        trigger_job(token, commit, "android-navigation-code-coverage-ci")
```

Comment:
it's not clear what happens if `publish_results == false`, the script trigger workflow **and** additionally triggered jobs, right?